src/main/resources/gui.yml

```diff
@@ -170,9 +170,12 @@ PlayerCommands:
 Commands:
   Item: bookshelf
   Metadata: 0
+Customs:
+  Item: writable_book
```

Comment:
This is not a valid Material name, I'm afraid the plugin is complaining whilst starting up.